package.yaml

```diff
@@ -0,0 +1,54 @@
+tasks:
+  - task: docker.Version
+    configFilePath: scripts/configure_dev-deps.sh
+
+  - task: shell.docker.Ensure
+    name: rpm
+    image: algorand/go-algorand-ci-linux-centos
+    version: '{{ docker.Version.outputs.version }}'
+    dockerFilePath: docker/build/cicd.centos.Dockerfile
+    dependencies: docker.Version
+
+  - task: shell.docker.Ensure
+    name: deb
+    image: algorand/go-algorand-ci-linux-ubuntu
+    version: '{{ docker.Version.outputs.version }}'
+    dockerFilePath: docker/build/cicd.ubuntu.Dockerfile
+    dependencies: docker.Version
+
+  - task: docker.Make
+    name: build
+    docker:
+      image: algorand/go-algorand-ci-linux-centos
+      version: '{{ docker.Version.outputs.version }}'
+      workDir: /projects/go-algorand
+    target: ci-build
+
+  - task: docker.Make
+    name: rpm
+    docker:
+      image: algorand/go-algorand-ci-linux-centos
+      version: '{{ docker.Version.outputs.version }}'
+      workDir: /projects/go-algorand
+      volumes: [ $HOME/go/bin:/projects/go-algorand/go/bin ]
+    target: package-rpm WORKDIR=/projects/go-algorand
+
+  - task: docker.Make
+    name: deb
+    docker:
+      image: algorand/go-algorand-ci-linux-ubuntu
+      version: '{{ docker.Version.outputs.version }}'
+      workDir: /projects/go-algorand
+      volumes: [ $HOME/go/bin:/projects/go-algorand/go/bin ]
+    target: package-deb WORKDIR=/projects/go-algorand
+
+jobs:
+  package:
+    configs:
+      arch: amd64
+    tasks:
+      - shell.docker.Ensure.rpm
+      - docker.Make.rpm
+      - shell.docker.Ensure.deb
+      - docker.Make.deb
```

Comment:
What about the packaging of the docker container?

Reply:
My thought was to have the docker container packaged in a separate task.  I think it makes sense to have that removed from the packaging of the deb and rpm.